tests/inc/class-wpseo-image-utils-test.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Yoast\WP\Free\Tests\Admin\Inc;
+
+use WPSEO_Image_Utils;
+use Yoast\WP\Free\Tests\Doubles\Inc\Image_Utils_Double;
+use Yoast\WP\Free\Tests\TestCase;
+use Brain\Monkey;
+
+/**
+ * Unit Test Class.
+ *
+ * @coversDefaultClass WPSEO_Image_Utils
+ *
+ * @group image-utils
+ */
+class Image_Utils_Test extends TestCase {
+
+	/**
+	 * Holds the instance of the class being tested.
+	 *
+	 * @var Image_Utils_Double
+	 */
+	private $instance;
+
+	/**
+	 * Set up the class which will be tested.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->instance = new Image_Utils_Double();
+	}
+
+	/**
+	 * Test whether the first image is returned from a term description.
+	 *
+	 * @covers ::get_first_content_image_for_term
+	 */
+	public function test_get_first_content_image_for_term() {
+
+		$term_descr =
+			'<p>This is a term description. It has several images:</p>
+			<img src=""/>
+			<img src="' . 'https://example.com/media/first_image.jpg' . '"/>
+			<img src="' . 'https://example.com/media/second_image.jpg' . '"/>
+			<p> That were all the images. Done! </p>';
+
+		Monkey\Functions\expect( 'term_description' )
+			->andReturn( $term_descr );
+
+		$first_image = $this->instance->get_first_content_image_for_term( '11' );
+
+		$expected = 'https://example.com/media/first_image.jpg';
+
+		$this->assertEquals( $expected, $first_image );
+	}
+
+	/**
+	 * Test whether null is returned when $term_id is null.
+	 *
+	 * @covers ::get_first_content_image_for_term
+	 */
+	public function test_get_first_content_image_for_term_when_null() {
+
+		$first_image = $this->instance->get_first_content_image_for_term( null );
+
+		$this->assertNull( $first_image );
+	}
+
+	/**
+	 * Test to run with a provider for the get_first_image method.
+	 *
+	 * @param mixed $images   The images to get first image from.
+	 * @param mixed $expected The expected value.
+	 * @param string $message The message to show when the test fails.
```

Comment:
Minor: these alignments could be improved.

Reply:
Done